index.bs

```diff
@@ -574,18 +575,15 @@ enum CookieSameSite {
   "none"
 };
 
-dictionary CookieStoreSetOptions {
+dictionary CookieInit {
```

Comment:
Hmmm.. wasn't the intent of the change to merge this with the CookieListItem dictionary, or was I mistaken?

Reply:
Oh I see, I think I misunderstood. The only difference between CookieListItem and CookieInit is that CookieInit does not have `secure`. So should this be something like this?
```
dictionary CookieInit {
  required USVString name;
  USVString value;
  USVString? domain = null;
  USVString path = "/";
  DOMTimeStamp? expires = null;
  CookieSameSite sameSite = "strict";
};

dictionary CookieListItem : CookieInit {
  boolean secure = true;
}

Reply:
That LGTM. Maybe the `secure` member wasn't apparent to kenneth. 

So once this all lands we may want to ping and say "here's what we ended up with" (but we can land and then ask)

Reply:
SG 👍 , thanks!